SPiDSDK/SPiDClient.m

```diff
@@ -262,7 +262,7 @@ - (NSString *)currentUserID {
 
 - (BOOL)isAuthorized {
     if (self.accessToken)
-        return YES;
+        return self.hasTokenExpired;
```

Comment:
We are authorized if:
1. We have an access token.
2. That token has expired

Shouldn't it be the negation of hasTokenExpired?